homeassistant/components/overkiz/__init__.py

```diff
@@ -64,9 +64,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
             client.get_places(),
         ]
         devices, scenarios, gateways, places = await asyncio.gather(*tasks)
-    except BadCredentialsException:
+    except BadCredentialsException as exception:
         _LOGGER.error("Invalid authentication")
```

Comment:
You shouldn't have to log this anymore. The exception will take care of that.
```suggestion
```

Reply:
Makes sense, I see that it indeed prints a warning. Would you say that `warning` is the best severity level for the other logging. We did use `error` since it is breaking, but perhaps this should be warning as well.

https://github.com/home-assistant/core/blob/60c7f49f6818459f584cfeae658242072332e881/homeassistant/components/overkiz/__init__.py#L67-L81

Reply:
Removed in 5242acf.